suite/tests/linux/bad-signal-stack.c

```diff
@@ -30,13 +30,15 @@
  * DAMAGE.
  */
 
-#include "tools.h"
 #include <unistd.h>
 #include <signal.h>
 #include <ucontext.h>
 #include <errno.h>
 #include <stdlib.h>
 #include <assert.h>
+/* i#6615 include tools.h after signal.h to avoid issues on
+ * ubuntu 22:04 caused by _GNU_SOURCE being defined */
+#include "tools.h"
```

Comment:
If we want to support building inside a container, could that be added to the automated tests?  Otherwise it is likely to be broken unknowingly in the future.

Reply:
I imagine going forward we will use containers more and more, especially for CI. We already do for our internal CI.

I'll raise this internally.